gateway/conf/nginx.conf.liquid

```diff
@@ -122,7 +122,32 @@ http {
 
     access_log {{ access_log_file | default: "/dev/stdout" }} time;
 
-    listen {{ port.apicast | default: 8080 }};
+    {%- assign http_port = port.apicast | default: 8080 %}
+    {%- assign https_port = env.APICAST_HTTPS_PORT %}
+
+    {% if http_port != https_port -%}
+      listen {{ http_port }};
+    {% endif %}
+
+    {% if https_port -%}
+    listen {{ https_port }} ssl;
+
+    {%- assign https_certificate = env.APICAST_HTTPS_CERTIFICATE -%}
+    ssl_certificate {% if https_certificate -%}
+      {{  https_certificate }}
```

Comment:
This one does not need ` | filesystem | first ` ?

Reply:
This is path that was given by user in ENV, so we should not need to do any transformations.

Reply:
Also, it does not really work. The `| filesystem` filter does not really work with absolute paths (yet).

Reply:
:+1: